frontend/tests/unit/models/census-tract-test.js

```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+//import { VARIABLE_MODE_LOOKUP } from 'labs-ceqr/app/models/census-tract';
+
+module('Unit | Model | census tract', function(hooks) {
+  setupTest(hooks);
+
+  test('it returns correct display mode from variable', function(assert) {
+    let store = this.owner.lookup('service:store');
+    let trans_total = store.createRecord('census-tract', { variable: 'trans_total' });
+    let trans_auto_total = store.createRecord('census-tract', { variable: 'trans_auto_total' });
+    let trans_public_total = store.createRecord('census-tract', { variable: 'trans_public_total' });
+
+    // Cannot get the Class constant import to work, so testing with magic strings
```

Comment:
`import { VARIABLE_MODE_LOOKUP } from 'labs-ceqr/app/models/census-tract';` should be 

`import { VARIABLE_MODE_LOOKUP } from 'labs-ceqr/models/census-tract';`

IDK why D:

Reply:
i thought i tried that too... will give it another go